limitador/src/limit.rs

```diff
@@ -26,8 +26,10 @@ impl From<String> for Namespace {
 #[derive(Eq, Debug, Clone, Serialize, Deserialize)]
 pub struct Limit {
     namespace: Namespace,
+    #[serde(skip_serializing)]
```

Comment:
Not sure why I changed that in the first PR, but this still needs to _read_ these values when reading the `LIMITS_FILE` obviously… otherwise all limits are `0` 🤦‍♂️